docs/source/configuration/authn-jwt.mdx

```diff
@@ -260,6 +266,18 @@ If you're using a key to perform JWT authentication with the Apollo Router, this
 </tbody>
 </table>
 
+### JWK Key Matching
+
+Some Identity Providers (IdPs) don't specify an `alg` or a `kid` for their keys. They will specify a `kty`, since that is mandatory in the JWKS specification..
+
+The router searches through the set of configured JWKSets to find the best matching JWK according to the following criteria:
+
+ 1. a matching `kid` and `alg`
```

Comment:
There's a fourth possibility if the JWT _doesn't_ include a `kid` and the JWK _does_ define its `alg` yeah?

1. Matching `kid` + `alg`
2. Matching `kid` + `kty`
3. Matching `alg`
4. Matching `kty`

Reply:
Yes. That's correct. I'll update the docs.

Reply:
done: 53b6295c1934a75d51e24fcb8f198576af8f7fb4